src/models/User.model.ts

```diff
@@ -3,6 +3,18 @@ import { model, Schema, Document } from "mongoose";
 export interface User extends Document {
   username: string;
   password: string;
+  name: string;
+  location: string;
```

Comment:
Is this field required?
```suggestion
  location?: string;
```